docs/workflow/testing/mono/testing.md

```diff
@@ -1,54 +1,83 @@
 # Running Tests using Mono Runtime
 
+Before running any tests, build mono in the way that you would like to test with.
+See the instructions for [Building Mono](../../building/mono/README.md)
+
 ## Running Runtime Tests
-We currently only support running tests against coreclr.  There are additional mono runtime tests in mono/mono, but they
-have not been moved over yet. Simply run the following command:
+* Build and run runtime tests against Mono JIT:
 
+    * Build and run one specific runtime test. From the root directory run the following command:
 ```
-dotnet build /t:RunCoreClrTests $(REPO_ROOT)/src/mono/mono.proj
+cd src/mono/netcore
+make run-tests-coreclr CoreClrTest="bash ../../artifacts/tests/coreclr/OSX.x64.Release/JIT/opt/InstructionCombining/DivToMul/DivToMul.sh"
 ```
 
-If you want to run individual tests, execute this command:
-
+    * Build and run all runtime tests. From the root directory run the following command:
 ```
-dotnet build /t:RunCoreClrTest /p:CoreClrTest="<TestName>" $(REPO_ROOT)/src/mono/mono.proj
+cd src/mono/netcore
+make run-tests-coreclr-all
 ```
 
-## Running Library Tests
-Running library tests against Mono is straightforward regardless of configuration.  Simply run the following commands:
+* Build and run runtime tests for Webassembly. From the root directory run the following command:
+```
+src/tests/build.sh -skipstressdependencies -excludemonofailures os Browser wasm <Release/Debug>
+```
+    * From the last few lines of the build log, you will see something like this
+```
+--------------------------------------------------
+ Example run.sh command
 
-1. Build and set the RuntimeFlavor
+ src/tests/run.sh --coreOverlayDir=<repo_root>artifacts/tests/coreclr/Browser.wasm.Release/Tests/Core_Root --testNativeBinDir=<repo_root>/artifacts/obj/coreclr/Browser.wasm.Release/tests --testRootDir=<repo_root>/artifacts/tests/coreclr/Browser.wasm.Release --copyNativeTestBin Release
+--------------------------------------------------
+```
+    * To run the tests, copy that command and add `wasm` at the end.
 
-```bash
-./build.sh /p:RuntimeFlavor=mono
+* Build and run Runtime Tests for Android x64. From the root directory run the following command:
```

Comment:
See previous sections for formatting suggestions. Same idea here.

Reply:
Fixed.